Monika After Story/game/script-topics.rpy

```diff
@@ -3587,6 +3587,35 @@ label monika_chloroform:
     m 1tsb "You're so cute when you're nervous."
     return
 
+init 5 python:
+    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
+      addEvent(Event(persistent.event_database,eventlabel="just_yuri",category=['club members'],prompt="Just Yuri mod",random=True))
+        
+label just_yuri:
+       m 1esc "[player]..."
```

Comment:
This should probably open with a casual opener such as "Hey, " rather than immediately calling the player's name.